plugins/basic/nexus-ui-extjs3-plugin/src/main/resources/static/js/Sonatype/utils.js

```diff
@@ -802,7 +802,6 @@ define('Sonatype/utils',['../extjs', 'Nexus/config', 'Nexus/util/Format', 'Sonat
             },
             failure : function(response, options) {
               Ext.namespace('Sonatype.utils').refreshTask.stop();
-              Ext.namespace('Sonatype.utils').clearCookie('JSESSIONID');
```

Comment:
clearing the cookie on failing to login does not make sense - you can't ask to login from the UI unless you are logged out anyways, so your session cookie is not valid anymore anyways